cmd/earthly/subcmd/build_cmd.go

```diff
@@ -500,21 +500,28 @@ func (a *Build) ActionBuildImp(cliCtx *cli.Context, flagArgs, nonFlagArgs []stri
 	}
 
 	cacheImports := make([]string, 0)
+	var cacheImportImageName string
 	if a.cli.Flags().RemoteCache != "" {
-		cacheImports = append(cacheImports, a.cli.Flags().RemoteCache)
+		cacheImportImageName, _, err = a.parseImageNameAndAttrs(a.cli.Flags().RemoteCache)
```

Comment:
What's the reason for discarding the attributes here? It would be better to raise an error saying attributes are not supported by 

What about error handling?

Reply:
From what I understand, in [buildkit](https://github.com/moby/buildkit?tab=readme-ov-file#output), there are only output keys and no input keys, which in Earthly terminology translates to export attributes without any corresponding import attributes. Consequently, these cannot be utilized in the import process without causing issues.

The critical aspect here is the --remote-cache flag, which facilitates the provision of a ref that is applicable to both exporting and importing. To address my concern, I should be able to specify attributes exclusively for the export process.

Additionally, I'm unclear about the import mechanism in Earthly. Specifically, I attempted to eliminate input attributes in the `solver.go` file, but this effort was unsuccessful. Earthly defaulted to using export attributes for the import process, which resulted in the malfunction of the import function.

Reply:
Sorry I misunderstood this change earlier, I didn't appreciate the fact that the import is _also_ set to the same `RemoteCache` value that's used for the export.

can you move the error handling to into this if statement -- currently the error handling is being run even when this code block is skipped.

In the case of imports, it might have been easier to just move `parseImageNameAndAttrs` into a utility package (and rename to `ParseImageNameAndAttrs`), which could be called both here and in the solver.go code.

Reply:
Sure, done.
@alexcb